src/components/RegisterForm.component/SellerRegistration.js

```diff
@@ -1,124 +1,151 @@
-import React from 'react'
-import { useForm } from 'react-hook-form'
+import React, {useState} from 'react'
 import './Form.css'
+import validation from "./Validation";
+import {useDispatch} from "react-redux";
+
+import {Link} from "react-router-dom";
 
 export const SellerRegistration = () => {
-  const {
-    register,
-    handleSubmit,
-    formState: { errors, isDirty, isValid },
+    const [values, setValues] = useState({
```

Comment:
lets rename this values from something for meaningful, same for setValues method